src/main/assets/scss/main.scss

```diff
@@ -358,8 +358,8 @@ div {
   flex-wrap: wrap;
 
   > div {
-    flex: 50%;
-    max-width: 50%;
+    flex: 33.33%;
```

Comment:
would the gov width overrides not work here?

Reply:
yes and no.
I can use `govuk-!-width-one-third` and that will display 3 in a row, however doesn't play well with loops, as if there are more elements they're just added in the row, they don't break down.